app/controllers/visitors_controller.rb

```diff
@@ -23,6 +25,13 @@ def update_holders
 
   private
 
+  def get_holder
```

Comment:
@raul-fuentes if we can remove get_holder call from index action maybe we can remove this method. This code is a little bit difficult to understand